src/content/docs/ai-monitoring/view-ai-data.mdx

```diff
@@ -0,0 +1,187 @@
+
+import aiTopleveAiResponsesSummary from 'images/ai_screenshot-crop_topleve-ai-responses-summary.webp'
+
+import aiResponseTable from 'images/ai_screenshot-crop_response-table.webp'
+
+import aiTraceViewAiResponse from 'images/ai_screenshot-full_trace-view-ai-response.webp'
+
+import aiTraceWaterfallPageSpanDetails from 'images/ai_screenshot-crop_trace-waterfall-page-span-details.webp'
+
+import aiTraceWaterfallPageErrorDetails from 'images/ai_screenshot-crop_trace-waterfall-page-error-details.webp'
+
+import aiAiModelComparisonPage from 'images/ai_screenshot-full_ai-model-comparison-page.webp'
+
+Enabling AI monitoring allows the agent to recognize and capture performance metrics and trace data about your app's AI-layer. You can track token usage, number of completions, and response time from your AI-powered app. When you see an error or an inaccurate response, scope to a trace-level view on a given prompt-response interaction to identify problems in the logic of your AI service. 
+
+<img
+    title="AI responses data"
+    alt="An image that shows the kind of data you get when you enable AI monitoring"
+    src={aiIntroAiUi}
+/>
+
+You can view your data by going to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring**. You can see your data from three different pages:
+
+* **AI responses**: Overview aggregated data from all your AI entities. Track your AI responses, times, and tokens, or see data about individual prompts and responses. 
+* **AI entities**: View a table summary of all entities reporting AI data. See entities with standard APM data like error rate, throughput, and app response time. When you select an entity, you can start exploring the APM **AI responses** page. 
+* **Compare models**: Compare token usage, response time, and error rate between different models. If you're conducting A/B tests, get all the information you need to make decisions about your AI-powered app.
+
+## AI responses page [#ai-responses]
+
+The top-level **AI responses** page shows your AI data in aggregate. Aggregated data takes the average total responses, response times, and token usage per response across all entities reporting AI data. On this page, response refers to an output from your AI-powered app when given a prompt. 
+
+If you own several apps with various implementations of different AI frameworks, you can get a general sense for how your AI models perform.  
+
+### Track total responses, average response time, and token usage  
+
+<img
+    title="AI responses response billboard and timeseries graphs"
+    alt="A cropped screenshot displaying the timeseries graphs and billboard info about AI data"
+    src={aiTimeseriesBillboard}
+/>
+
+The three tiles show general performance metrics about your AI's responses. These tiles may not tell you the exact cause behind a problem, but they're useful for identifying anomalous fluctuations in your app's performance. 
```

Comment:
"exact cause of the problem", I think. Also can we change "anomalous fluctuations" to just "anomalies"?

Reply:
"anomalous fluctuations" is such a funny phrase that i can't believe i wrote lol